core/controlplane/config/templates/cluster.yaml

```diff
@@ -174,8 +174,29 @@ worker:
 #          # Must be within the range between 100 and 2000
 #          # Defaults to worker.spotFleet.unitRootVolumeIOPS * weightedCapacity if omitted
 #          #rootVolumeIOPS:
+#
+#        #  # Additional EBS volumes mounted on the worker
+#        #  # No additional EBS volume by default. All parameter values do not default - they must be explicitly defined
+#        #  ebsVolumes:
```

Comment:
How about changing this to `volumeMounts:`?

In my mental model:
* `type` and `iops`, `size` are properties belonging to an "EBS volume`
* `device` is a property belonging to an "EBS volume attachment"
* `mountPath` is a property belonging to an "EBS volume mount"

respectively and therefore I took it as a short-hand definition of a set of a volume & an attachment & a mount rather than just a volume.